x/chainlet/keeper/ccv.go

```diff
@@ -41,15 +41,13 @@ func (k *Keeper) addConsumer(ctx sdk.Context, chainId string, spawnTime time.Tim
 	k.providerKeeper.AppendPendingVSCPackets(ctx, chainId, packet)
 	k.providerKeeper.IncrementValidatorSetUpdateId(ctx)
 
-	// Send it right after a channel is created
-	k.setPendingVSC(ctx, chainId)
-
+	k.setPendingInit(ctx, chainId)
 	return nil
 }
 
 // Forces sending queued VSC packets of new chainlets without waiting for the the provider epoch to end.
-func (k *Keeper) ForcePendingVSC(ctx sdk.Context) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.ChainletPendingVSCKey)
+func (k *Keeper) InitConsumers(ctx sdk.Context) {
```

Comment:
what is VSC? It's commonly used here in the package but what's the abbreviation for?